core/src/test/scala/org/apache/spark/rdd/RDDSuite.scala

```diff
@@ -897,4 +897,23 @@ class RDDSuite extends FunSuite with SharedSparkContext {
       mutableDependencies += dep
     }
   }
+
+  test("Nested RDDs are not supported (SPARK-5063)") {
```

Comment:
a nit pick: i don't think we have a standard, but so far test case names start with lower case.